main.cpp

```diff
@@ -245,19 +288,30 @@ struct Speakers
     int size = 12;
     float freqRange = 145.4f;
     float impedence = 50.0f;
+    Speakers();
     void produceAudio();
     void receiveSignal();
     void vibrateCone();
 };
 
+Speakers::Speakers()
+{
+    brand = "";
+    modelNum = 0;
+    size = 0;
+    freqRange = 0;
+    impedence = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise